packages/apps/composer-app/src/components/Main.tsx

```diff
@@ -4,7 +4,7 @@
 
 import React, { ComponentProps } from 'react';
 
-import { mx } from '@dxos/react-uikit';
+import { mx } from '@dxos/react-ui';
```

Comment:
Will this also come from appkit? My assumption based on what we discussed yesterday was that outside of halo, the apps shouldn't need to access the OS patterns directly once a steady state is reached

Reply:
No, `mx` needs to come from `react-components` because that is where the Tailwind stack is defined.

Reply:
Or, do you mean `Main`? I think since it isn't about something DXOS is doing, it's probably best for apps to define a component like that on their own. The only component in appkit that isn't about what DXOS is doing (at the moment) is `AppLayout`, though that may later come in handy when we have OS patterns for apps that want to coordinate layout between their content and OS panels.